src/Subscriber/CategoryView.php

```diff
@@ -51,11 +51,24 @@ public function onPageLoaded(NavigationPageLoadedEvent $event): void
             [
                 'communication' => [
                     'search-immediate' => !$isHome && !$disableImmediate ? 'true' : 'false',
-                    'add-params'       => $path ? implode(',', $this->initial + [sprintf('filter=%s', urlencode($this->fieldName . ':' . $path))]) : '',
+                    'add-params'       => $this->prepareAddParams($path),
+                    'category-page'    => $this->prepareCategoryPath($path),
                 ],
             ]);
     }
 
+    private function prepareAddParams($path): string
```

Comment:
I think if we need to to leave add-params, we should render here only that, what has been defined as $this->initial